lib/components/PendingMultisig.js

```diff
@@ -0,0 +1,40 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { Header, Table } from '@bpanel/bpanel-ui';
+
+class PendingMultisig extends PureComponent {
```

Comment:
I'm not sure which way is best myself, I could see the argument for either, but do you think it makes to switch this component to be called `PendingMultiparty` as well to match what the UI displays?

Reply:
What do you mean? Ideally the UI doesn't change if at all once we've stabilized. Just thinking about the situation where someone wants to make a change to the `PendingMultiparty` component and can't find it immediately. It's not hard to deduce of course, but it's the convention we use with virtually every other component right?